src/pages/mypage/profile/page.tsx

```diff
@@ -0,0 +1,107 @@
+// import React from 'react';
+
+import { useState } from 'react';
+
+export default function ProfilePage() {
+  // 자치기구 여부 확인 -> api 완료되면 수정예정
+  const [isAssociation] = useState(false);
+  return (
+    <main>
+      {/* 유저 정보 컨테이너 (자치기구, 일반학우) */}
+      {isAssociation ? (
+        <>
+          <div className="border-#D9D9D9 mb-16 ml-16 mr-10 mt-16 flex flex-row items-center rounded-lg border-2 bg-white p-10">
+            <img className="mr-8 h-28 w-28" src="/image/mypage/profile_img.png" alt="profile_default_img" />
+            <div>
+              <div className="mb-4">
+                <span className="text-xl font-extrabold">제 19대 글로벌미디어학부 학생회 미리내</span>
+              </div>
+              <div className="mb-1">
+                <span>IT대학 • </span>
+                <span>글로벌미디어학부</span>
+              </div>
+            </div>
+          </div>
+          <div className="mb-8 pl-16">
+            <h3 className="text-lg font-bold">기본정보</h3>
+            <div className="p-6 text-sm">
```

Comment:
이 div 내부처럼 자식 사이의 일정한 마진을 주고 싶을 때에는 `flex`주고 `gap` 속성을 사용하는게 전 더 편하더라구요

```suggestion
            <div className="flex flex-col gap-4 mb-4 p-6 text-sm">
```

요렇게 작성하고 자식들의 `mb-4`를 없애주면 개인적으론 더 깔끔해 보이지 않을까 싶습니다!

Reply:
시간내서 자세하게 리뷰 해주셔서 감사합니당 ㅠ ㅠ 다 반영해서 작업 하겠습니다 ! 체고 🥹